pkgs/tools/networking/bitmask-vpn/default.nix

```diff
@@ -13,32 +13,29 @@
 , iproute2
 , iptables
 , procps
-, qmltermwidget
 , qtbase
 , qtdeclarative
-, qtgraphicaleffects
-, qtinstaller
-, qtquickcontrols
-, qtquickcontrols2
+, qtsvg
 , qttools
+, qtwayland
 , CoreFoundation
 , Security
 , provider ? "riseup"
 }:
 let
-  version = "0.21.11";
+  version = "git";
```

Comment:
https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming

Reply:
I have updated with the correct naming convention, thanks